instrumentation/logback/logback-appender-1.0/javaagent/build.gradle.kts

```
@@ -54,4 +54,5 @@ tasks.withType<Test>().configureEach {
   // TODO run tests both with and without experimental log attributes
   jvmArgs("-Dotel.instrumentation.logback-appender.experimental.capture-mdc-attributes=*")
   jvmArgs("-Dotel.instrumentation.logback-appender.experimental-log-attributes=true")
+  jvmArgs("-Dotel.instrumentation.logback-appender.experimental.code-attributes=true")
```

(to mimic `.experimental.capture-mdc-attributes` name)
```suggestion
  jvmArgs("-Dotel.instrumentation.logback-appender.experimental.capture-code-attributes=true")
```

---

Done for implementation and testing